selfdrive/sensord/sensors_qcom2.cc

```diff
@@ -207,16 +168,23 @@ int sensor_loop() {
     std::this_thread::sleep_for(std::chrono::milliseconds(10) - (end - begin));
   }
 
-  for (Sensor *sensor :  sensors) {
+  interrupt_thread.join();
+
+  for (auto [sensor, _] : all_sensors) {
```

Comment:
running it on all sensors, would trigger a could not shutdown bmx sensor message, on non bmx
devices.